receiver/mongodbatlasreceiver/metadata.yaml

```diff
@@ -0,0 +1,470 @@
+  memory_state:
+    description: Memory usage type
+    enum:
+      - resident
+      - virtual
+      - mapped
+      - computed
+  direction:
+    description: Network traffic direction
+    enum:
+      - receive
+      - transmit
+  storage_status:
+    description: Views on database size
+    enum:
+      - total
+      - data_size
+      - index_size
+      - data_size_wo_system
+  operation:
+    description: Type of database operation
+    enum:
+      - cmd
+      - query
+      - update
+      - delete
+      - getmore
+      - insert
+  cluster_role:
+    description: Whether process is acting as replica or primary
+    enum:
+      - primary
+      - replica
+  document_status:
+    description: Status of documents in the database
+    enum:
+      - returned
+      - inserted
+      - updated
+      - deleted
+  execution_type:
+    description: Type of command
+    enum:
+      - reads
+      - writes
+      - commands
+  scanned_type:
+    description: Objects or indexes scanned during query
+    enum:
+      - index_items
+      - objects
+  disk_direction:
+    description: Measurement type for disk operation
+    enum:
+      - read
+      - write
+      - total
+  disk_status:
+    description: Disk measurement type
+    enum:
+      - free
+      - used
+  object_type:
+    description: MongoDB object type
+    enum:
+      - collection
+      - index
+      - extent
+      - object
+      - view
+      - storage
+      - data
+  oplog_type:
+    description: Oplog type
+    enum:
+      - slave_lag_master_time
+      - master_time
+      - master_lag_time_diff
+
+metrics:
+  mongodbatlas.process.asserts:
+    description: Number of assertions
+    extended_documentation: Aggregates MongoDB Metrics ASSERT_REGULAR, ASSERT_USER, ASSERT_MSG, ASSERT_WARNING
+    unit: "{assertions}/s"
+    labels: [assert_type]
+    data:
+      type: gauge
+  mongodbatlas.process.background_flush:
+    description: Amount of data flushed in the background
+    extended_documentation: MongoDB Metric BACKGROUND_FLUSH_AVG
+    unit: 1
+    data:
+      type: gauge
+  mongodbatlas.process.cache.io:
+    description: Cache throughput (per second)
+    extended_documentation: Aggregates MongoDB Metrics CACHE_BYTES_READ_INTO, CACHE_BYTES_WRITTEN_FROM
+    unit: By
+    labels: [cache_direction]
+    data:
+      type: gauge
+  mongodbatlas.process.cache.size:
+    description: Cache sizes
+    extended_documentation: Aggregates MongoDB Metrics CACHE_USED_BYTES, CACHE_DIRTY_BYTES
+    unit: By
+    labels: [cache_status]
+    data:
+      type: gauge
+  mongodbatlas.process.connections:
+    description: Number of current connections
+    extended_documentation: MongoDB Metric CONNECTIONS
+    unit: "{connections}"
+    data:
+      type: gauge
+  mongodbatlas.process.cpu.usage:
+    description: CPU Usage (%)
+    extended_documentation: Aggregates MongoDB Metrics PROCESS_CPU_KERNEL, PROCESS_CPU_USER, MAX_PROCESS_CPU_KERNEL, MAX_PROCESS_CPU_USER
```

Comment:
Why adding a "max" in the list here? I think you need to have 2 metrics (the rule is to not merge different aggregations) so you will get `mongodbatlas.process.cpu.usage.average` `mongodbatlas.process.cpu.usage.max`

Reply:
Separated these out. Thanks, I didn't know that rule.